tests/test_task.py

```diff
@@ -0,0 +1,132 @@
+"""
+Task Test Suite
+"""
+
+import unittest
+
+from pydantic import Field, ValidationInfo, field_validator
+
+import aind_behavior_curriculum as abc
+
+
+class ExampleTask(abc.Task):
+    """
+    Example Task
+    """
+
+    # Required: Define type annotations for strict type checks.
+    # Optional: Make fields immutable with the 'frozen=True'.
+    field_1: int = Field(0)
```

Comment:
Default values should be defined using the `default` keyword. Intellisense uses this keyword to provide extra information to users.
e.g:

```python
field_1: int = Field(default=0)
```